src/router/routes.ts

```diff
@@ -44,19 +54,25 @@ export const routes = [
     }
   },
   {
-    path: '/limitedRestorationToFull',
-    name: RouteNames.RESTORATION_CONVERSION,
-    component: Restoration,
+    path: '/limitedRestorationExtension',
+    name: RouteNames.RESTORATION_EXTENSION,
+    component: LimitedRestorationExtension,
+    props: route => ({
+      restorationId: route.query['restoration-id']
+    }),
     meta: {
       requiresAuth: true,
       isStaffOnly: true,
       filingType: FilingTypes.RESTORATION
     }
   },
   {
-    path: '/limitedRestorationExtension',
-    name: RouteNames.RESTORATION_EXTENSION,
-    component: Restoration,
+    path: '/limitedRestorationToFull',
+    name: RouteNames.RESTORATION_CONVERSION,
+    component: LimitedRestorationToFull,
+    props: route => ({
+      restorationId: route.query['restoration-id']
+    }),
```

Comment:
Passing the restoration-id into the view as a prop.

Reply:
I think I remember a refactoring ticket talking about this. Please add a comment to that ticket indicating what you have already done and maybe pointing to this code as an example for other filings.

Reply:
I don't think there's a refactoring ticket to use props to pass route parameters to the views.  Shall I create one?  It's a good practice.

Reply:
Sure, create one.

Reply:
Done:  /bcgov/entity#15708

Reply:
I'm happy with this conversation now.